web/app/scripts/src/main.es.js

```diff
@@ -1,4 +1,8 @@
-/* global console */
-// TODO: remove the following two lines
 import { angular } from 'libraries';
-/* eslint no-console: 0 */
+
+import Api from 'services/api';
+import ProjectConfig from 'providers/project-config';
+
+angular.module('incuna-surveys', [])
+    .provider('ProjectConfig', ProjectConfig)
+    .service('Api', Api);
```

Comment:
We usually name services as `ApiService`, like we do `FooCtrl` or `FooController`, but is no biggy